app/src/main/res/layout/hints_and_solution_fragment.xml

```diff
@@ -33,24 +44,23 @@
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       android:background="@color/hintsAndSolutionBackground"
-      app:layout_constraintTop_toBottomOf="@+id/hints_and_solution_toolbar">
+      app:layout_constraintTop_toBottomOf="@+id/hints_and_solution_app_bar_layout">
 
       <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/hints_and_solution_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
-        android:paddingTop="8dp"
-        android:paddingBottom="128dp"
+        android:paddingTop="20dp"
```

Comment:
Top padding when added with item content padding is not matching

Reply:
I removed margin top from the item and added bottom margin to divider to match.